sled-agent/src/illumos/dladm.rs

```diff
@@ -13,6 +14,17 @@ use std::str::FromStr;
 pub const VNIC_PREFIX: &str = "ox";
```

Comment:
Is this constant still used? or is the only VNIC type with the `ox` prefix just the `oxControl` type?

Reply:
There is no longer an `oxGuest` prefix, that's true. I opted to name the guest VNICs after the xde devices they're sitting on top of: `opteN -> vopteN`. It does look like the prefix is still used, at `vnic.rs:60`, `vnic.rs:77`, and `vnic.rs:98`, though those probably could be simplified. Or rather the "Control" part of the prefix could go away. So we'd have all control VNICs with names like `ox<SCOPE>`, e.g., `oxInstance` for the propolis links or `oxStorage` for the Crucible links. Is that better?

Reply:
Eh, we don't need to change the naming scheme in this PR - guess I was mostly validating that this particular constant could become redundant, since all "ox" vnics would be "oxControl" vnics.

Anyway, if it's used elsewhere let's punt on this - getting the fixes in from this PR are more important than these refactors.